### ISSUE TITLE
Support to asynchronous notification of events?

### ISSUE THREAD

**Issue by dbonino** [intel-iot-devkit/tinyb#29]
Is it possible to add, in Java bindings, support to notification events generated by the Bluez stack when a device notifies a new characteristic value? E.g, by offering an interface to register a listener to Bluetooth Events generated by the Bluez stack when devices are notifying...

**Comment by petreeftime** [comment]
It's a feature I want to include in the next version, but have not yet started work on it.

**Comment by dbonino** [comment]
That will be really useful, thanks

**Comment by youcangetme** [comment]
I just got to this point in my code and spent the morning tracking BluetoothCallback and BluetoothEvent objects thinking those would lead me somewhere. The Bluetooth hardware I developed is sending notifications and it would be much easier to let the Java side respond to those notifications as oppose to reading in a loop.

So development doesn't grind to a halt I will do the loop and read thing but count me in as needing this feature ASAP.
